src/utils/__tests__/getServerRuntimeConfig.test.ts

```diff
@@ -8,7 +8,6 @@ const serverRuntimeConfig = {
   oidcClientSecret: 'secret',
   oidcIssuer: 'https://tunnistamo-backend:8000',
```

Comment:
Perhaps Tunnistamo references should be removed from here too (another affected key in this file is `oidcApiTokensUrl`).

Reply:
Changed